src/MongoDB/WriteResult.c

```diff
@@ -50,6 +50,108 @@ PHONGO_API zend_class_entry *php_phongo_writeresult_ce;
 
 zend_object_handlers php_phongo_handler_writeresult;
 
+static bool php_phongo_writeresult_get_writeconcernerror(php_phongo_writeresult_t *intern, zval *return_value TSRMLS_DC)
```

Comment:
Although these two functions return a boolean indicating their success, we don't actually check for it later. The return value is always initialized to something usable (e.g. empty array or null), so we add it to the debug info's hash regardless. If an exception was thrown, it will be caught above the debug handler (technically get_properties).